Gemfile

```diff
@@ -35,6 +35,8 @@ gem "paper_trail"
 gem "rolify", "~> 6.0"
 # Enforces "safe" migrations.
 gem "strong_migrations", "1.4.1"
+# imports (used to set up counties)
+gem "activerecord-import"
```

Comment:
Rails supports mass inserts now: https://blog.saeloun.com/2022/07/26/rails-6-insert-all.html

Reply:
Alrighty.  That's the last thing I'm adding in the "low hanging fruit" push I'll be doing shortly.